docs/API_DESIGN.md

```diff
@@ -2,14 +2,15 @@
 
 **Last Updated** May 30, 2024
 
-**Version** 0.1.0
```

Comment:
sure you dont want to save 1.0.0 for the for sure for sure globalish release?

Reply:
hehe I was hoping someone would see this

no

🙂 I want to set aside marketing material and _**BUILD**_